backend/run.ts

```diff
@@ -1,10 +1,10 @@
-import process from "process";
-import open from "open";
+import process from 'process';
+import open from 'open';
```

Comment:
it seems you used prettier which caused some noise in the PR, we should integrate prettier to avoid such styling issues